src/main/resources/db/migration/core/V1.1__ADDRESS.sql

```diff
@@ -1,11 +1,3 @@
-CREATE TABLE client
-(
-    id           VARCHAR(50)  NOT NULL PRIMARY KEY,
-    name         VARCHAR(100) NOT NULL,
-    address_id   VARCHAR(50)  NOT NULL,
-    created_date TIMESTAMP
-);
-
 CREATE TABLE address
```

Comment:
The reason I used the name of the file with `V1.0.000` is to say that this is the script 000 from the version 1.0.

When we have a new version, like 1.1, we could have the script V1.1.123.

Meaning just by the script name we now when this script was added.


The same approach also works with V1.1, V1.2, but we lose the minor version on it.

What do you think?

Reply:
Will switch to 1.00